crates/bls-crypto/src/hash_to_curve/try_and_increment.rs

```diff
@@ -219,9 +388,10 @@ mod test {
             "b68e1db4b648801676a79ac199eaf003757bf2a96cdbb804bfefe0484afdc0cc299d50d660221d1de374e92c44291200",
         ].into_iter().map(|x| hex::decode(&x).unwrap()).collect::<Vec<_>>();
 
-        test_hash_to_group(&*COMPOSITE_HASH_TO_G1, &mut rng, expected_hashes)
+        super::test::test_hash_to_group(&*COMPOSITE_HASH_TO_G1, &mut rng, expected_hashes)
     }
 
+    #[cfg(not(feature = "compat"))]
```

Comment:
Is this needed? We're already in a `#[cfg(all(test, not(feature = "compat")))]` module

Reply:
Not needed, mistake